guides/nuxtjs-starter-guide/index.mdx

```diff
@@ -0,0 +1,30 @@
+---
+_schema: guide-item
+_uuid: "af4eccad-9425-4d97-9c3d-3325b7699d41"
+_created_at: "2023-06-22 06:54:28 +0000"
+title: "Introduction"
+nav_title: "Introduction"
+order: 1
+published: true
+image: "/documentation/static/CloudCannonDocumentationog.jpg"
+description: "This guide will walk through the steps required to get your Nuxt site built, editable and live on CloudCannon."
+tags: []
+related_articles: null
+related_links: null
+guide_listing:
+  show_on_index: true
+  category: Starter Guides
+---
+
+Nuxt is a popular web framework that lets you build websites and applications with Vue.js. Nuxt has comprehensive tooling allowing it to be used as an SSG to create fully static sites.
+CloudCannon makes it easy to store your content in your Git Repository and have non-developers update it. There's no need to teach people about front matter, markdown or Git.
+
+To get the best and "most easy to mesh with cloudcannon" experience, we recommend that you build your Nuxt site with the [Nuxt Content](https://content.nuxtjs.org/) module.
```

Comment:
```suggestion
To get the best and "easiest to mesh with Cloudcannon" experience, we recommend that you build your Nuxt site with the [Nuxt Content](https://content.nuxtjs.org/) module.
```

Reply:
Ooops that was placeholder copy that I forgot to replace 😅